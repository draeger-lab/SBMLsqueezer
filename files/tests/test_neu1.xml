<?xml version="1.0" encoding="UTF-8"?>
<sbml xmlns="http://www.sbml.org/sbml/level2" level="2" version="1">
  <model id="untitled">
    <listOfUnitDefinitions>
      <unitDefinition id="substance" name="Predefined unit substance">
        <listOfUnits>
          <unit kind="mole" scale="-3"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="volume" name="Predefined unit volume">
        <listOfUnits>
          <unit kind="litre" scale="-3"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="area" name="Predefined unit area">
        <listOfUnits>
          <unit kind="metre" exponent="2"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="length" name="Predefined unit length">
        <listOfUnits>
          <unit kind="metre"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="time" name="Predefined unit time">
        <listOfUnits>
          <unit kind="second"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="mmolePerSecond">
        <listOfUnits>
          <unit kind="mole" scale="-3"/>
          <unit kind="second" exponent="-1"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="mM">
        <listOfUnits>
          <unit kind="mole" scale="-3"/>
          <unit kind="litre" exponent="-1"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="mmole_per_second">
        <listOfUnits>
          <unit kind="mole" scale="-3"/>
          <unit kind="second" exponent="-1"/>
        </listOfUnits>
      </unitDefinition>
      <unitDefinition id="per_kmole_per_second">
        <listOfUnits>
          <unit kind="mole" exponent="-1" scale="3"/>
          <unit kind="second" exponent="-1"/>
        </listOfUnits>
      </unitDefinition>
    </listOfUnitDefinitions>
    <listOfCompartments>
      <compartment id="default" size="1" units="volume"/>
    </listOfCompartments>
    <listOfSpecies>
      <species id="s1" name="s1" compartment="default" initialAmount="0" substanceUnits="substance"/>
      <species id="s2" name="s2" compartment="default" initialAmount="0" substanceUnits="substance"/>
      <species id="s3" name="s3" compartment="default" initialAmount="0" substanceUnits="substance"/>
    </listOfSpecies>
    <listOfReactions>
      <reaction id="re1" reversible="false">
        <listOfReactants>
          <speciesReference species="s1"/>
          <speciesReference species="s2"/>
        </listOfReactants>
        <listOfProducts>
          <speciesReference species="s3"/>
        </listOfProducts>
        <kineticLaw>
          <math xmlns="http://www.w3.org/1998/Math/MathML">
            <apply>
              <times/>
              <ci> kass_re1 </ci>
              <ci> s1 </ci>
              <ci> default </ci>
              <ci> s2 </ci>
              <ci> default </ci>
            </apply>
          </math>
          <listOfParameters>
            <parameter id="kass_re1" name="Association constant of reaction re1" value="1" units="per_kmole_per_second"/>
          </listOfParameters>
        </kineticLaw>
      </reaction>
    </listOfReactions>
  </model>
</sbml>
