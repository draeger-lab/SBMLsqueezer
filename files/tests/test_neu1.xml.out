\BOOKMARK [1][]{section.1}{1 General Overview}{}
\BOOKMARK [1][]{section.2}{2 Unit Definitions}{}
\BOOKMARK [2][]{subsection.2.1}{2.1 Unit micromole}{section.2}
\BOOKMARK [2][]{subsection.2.2}{2.2 Unit molecules}{section.2}
\BOOKMARK [2][]{subsection.2.3}{2.3 Unit um2}{section.2}
\BOOKMARK [2][]{subsection.2.4}{2.4 Unit molecules\137um\1372\137s\1371}{section.2}
\BOOKMARK [2][]{subsection.2.5}{2.5 Unit s\1371}{section.2}
\BOOKMARK [2][]{subsection.2.6}{2.6 Unit molecules\137um\1372}{section.2}
\BOOKMARK [2][]{subsection.2.7}{2.7 Unit um2\137molecules\1371\137s\1371}{section.2}
\BOOKMARK [2][]{subsection.2.8}{2.8 Unit uM\1371\137s\1371}{section.2}
\BOOKMARK [2][]{subsection.2.9}{2.9 Unit uM}{section.2}
\BOOKMARK [2][]{subsection.2.10}{2.10 Unit substance}{section.2}
\BOOKMARK [2][]{subsection.2.11}{2.11 Unit volume}{section.2}
\BOOKMARK [2][]{subsection.2.12}{2.12 Unit area}{section.2}
\BOOKMARK [2][]{subsection.2.13}{2.13 Unit length}{section.2}
\BOOKMARK [2][]{subsection.2.14}{2.14 Unit time}{section.2}
\BOOKMARK [1][]{section.3}{3 Compartments}{}
\BOOKMARK [2][]{subsection.3.1}{3.1 Compartment PM}{section.3}
\BOOKMARK [2][]{subsection.3.2}{3.2 Compartment Cytosol}{section.3}
\BOOKMARK [1][]{section.4}{4 Species}{}
\BOOKMARK [1][]{section.5}{5 Parameters}{}
\BOOKMARK [1][]{section.6}{6 Rules}{}
\BOOKMARK [2][]{subsection.6.1}{6.1 Rule Ratebasal\137PIPsyn}{section.6}
\BOOKMARK [2][]{subsection.6.2}{6.2 Rule Ratestim\137PIPsyn}{section.6}
\BOOKMARK [2][]{subsection.6.3}{6.3 Rule signal}{section.6}
\BOOKMARK [2][]{subsection.6.4}{6.4 Rule kr\137PIP2PH}{section.6}
\BOOKMARK [2][]{subsection.6.5}{6.5 Rule Rate\137PIP2Synbasal}{section.6}
\BOOKMARK [2][]{subsection.6.6}{6.6 Rule Rate\137PIP2SynStim}{section.6}
\BOOKMARK [2][]{subsection.6.7}{6.7 Rule kr\137IP3PH}{section.6}
\BOOKMARK [1][]{section.7}{7 Events}{}
\BOOKMARK [2][]{subsection.7.1}{7.1 Event event\1370000001}{section.7}
\BOOKMARK [2][]{subsection.7.2}{7.2 Event event\1370000002}{section.7}
\BOOKMARK [2][]{subsection.7.3}{7.3 Event event\1370000003}{section.7}
\BOOKMARK [2][]{subsection.7.4}{7.4 Event event\1370000004}{section.7}
\BOOKMARK [2][]{subsection.7.5}{7.5 Event event\1370000005}{section.7}
\BOOKMARK [1][]{section.8}{8 Reactions}{}
\BOOKMARK [2][]{subsection.8.1}{8.1 Reaction PIPSyn}{section.8}
